a11y_pygments/a11y_dark/README.md

```diff
@@ -1,8 +1,10 @@
-# A11y dark
+# A11Y Dark
```

Comment:
Is the upper case here on purpose ? No objections, just unsure.

Reply:
Yeah in [line 62 of the script](https://github.com/Quansight-Labs/accessible-pygments/pull/42/files#diff-e976ce7b71b94a38fbe6e41fc1afcd62989c8ee279fbe9345d15f7f4543547edR62), I start with the theme module name (here, "a11y_dark"), replace the underscores with spaces ("a11y dark") and title-case the theme name ("A11Y Dark")